--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@
 #
 set(JSONCPP_OLDEST_VALIDATED_POLICIES_VERSION "3.8.0")
 set(JSONCPP_NEWEST_VALIDATED_POLICIES_VERSION "3.13.2")
-cmake_minimum_required(VERSION ${JSONCPP_OLDEST_VALIDATED_POLICIES_VERSION})
+cmake_minimum_required(VERSION 3.10)
 if("${CMAKE_VERSION}" VERSION_LESS "${JSONCPP_NEWEST_VALIDATED_POLICIES_VERSION}")
     #Set and use the newest available cmake policies that are validated to work
     set(JSONCPP_CMAKE_POLICY_VERSION "${CMAKE_VERSION}")